domain/error/error_domain_v3.0.0.yaml

```diff
@@ -63,7 +63,7 @@ components:
           type: string
           maxLength: 100
           example: 'invalidData'
-        erorCodeMessage:
+        errorCodeMessage:
```

Comment:
This is super input - unfortunately we have published this API - so it cannot be modified :-( It would have to be fixed in a 3.0.1 (the description typos...)
With regards to the typo in the `errorCodeMessage` this is just super unfortunate - we would have to fix this in a Major release (or deprecate the field and add a CORRECTLY spelled property next to it...
DT-APIs are still in Beta - so those I can fix. OVS is a bigger problem :-(

Reply:
We have published the Error Domain...

Reply:
understood, cleaned the commit up apologies !